waiter/config-full.edn

```diff
@@ -830,6 +830,14 @@
             :not-found "Not found"
             :prestashed-tickets-not-available "Prestashed jobsystem tickets not available"}
 
+ ;; Waiter launches a separate thread pool executor to perform blocking reads on incoming request bodies
+ :stream-reader {;; the maximum number of concurrent input stream reads allowed in the executor (default 150)
+                 :concurrency-level 150
+                 ;; the idle time before cached threads from the executor are terminated (default 5)
+                 :keep-alive-mins 5
+                 ;; the number of concurrent requests that are supported by the executor (default 50000)
+                 :queue-limit 50000}
+
```

Comment:
Should `:stream-reader` be a sub-map under `:server-options`?

Reply:
The `server-options` are used to initialize the Waiter jetty server. The `stream-reader` is more the client part that is streaming data to the backend, than the server part which is receiving requests.